manifest.json

```diff
@@ -1105,6 +1105,13 @@
       "id": "smallclock",
       "mod_version": "3"
     },
+    {
+      "description": "Opens the selected filename or path in project. Useful to open imports.",
+      "version": "0.1",
+      "path": "plugins/smartopenselected.lua",
+      "id": "smartopenselected",
+      "mod_version": "3"
+    },
```

Comment:
If you want to, you can add a `name` parameter for a "normal", user-visible name.